resource_planning/models/resource_resource.py

```diff
@@ -68,18 +69,15 @@ def _check_user_type_fields(self):
 
     @api.multi
     def action_unavailable(self):
-        for resource in self:
-            resource.state = "unavailable"
+        self.write({"state": "unavailable"})
```

Comment:
What is the reason for using `write` instead of replacing the attributes ?

Reply:
It's unrelated _en passant_ refactoring indeed. Avoiding the loops sends one query to the db for all records